bootstrap_cfn/errors.py

```diff
@@ -77,3 +77,33 @@ def __init__(self, autoscaling_group, expected_instance_count, instances):
             "Could not find {} instances in autoscaling group {}. Actual state is {} instances, {}"
             .format(expected_instance_count, autoscaling_group, len(instances), instances)
         )
+
+
+class TagRecordExistConflictError(BootstrapCfnError):
+    def __init__(self, stack_tag):
+        msg = ("An {} record already exists. ".format(stack_tag))
+        super(TagRecordExistConflictError, self).__init__(msg)
+
+
+class ActiveTagExistConflictError(BootstrapCfnError):
+    def __init__(self, stack_id):
+        msg = ("An active record already exists in. ".format(stack_id))
```

Comment:
The `format` parameter is not used. Presumably something like the following is intended:

```python
msg = 'An active record already exists in {}.'.format(stack_id)
```